zulipterminal/ui_tools/views.py

```diff
@@ -1605,7 +1621,21 @@ def __init__(
             msg_info[1][1].append(("Edit History", keys))
         # Render the category using the existing table methods if links exist.
         if message_links:
-            msg_info.append(("Message Links", []))
+            interact_with_link_keys = "[{}]".format(
+                ", ".join(map(str, keys_for_command("ENTER")))
+            )
+            msg_info.append(
+                (
+                    (
+                        "Message Links",
+                        f"Press {interact_with_link_keys} to interact with links",
```

Comment:
This change is distinct from the change to the category feature that supports it - or to put it differently: can this specific change be included only with similar changes for the other categories which may appear?

This feature can also be applied to other popups, though we may want to discuss that first.

Reply:
I'm not sure if we discussed this in the stream, but I was referring to having this in a different commit to the one which adds category heading support.